src/styles/burger.scss

```
@@ -0,0 +1,97 @@
+
+@media screen and (max-width: $small-screen) {
+  .burger {
+    display: block;
+    
+    width: 18px;
+    height: 12px;
+    
+    background-size: contain;
+    background-position: center;
+    background-repeat: no-repeat;
+    z-index: 2;
+    
+    label[for="nav-trigger"] {
+      display: block;
+      width: 18px;
+      height: 12px;
+      background-image: url("/images/burger.png");
+      background-size: contain;
+      background-position: center;
+      background-repeat: no-repeat;

```

коли кілька параметрів можна скоротити:
```suggestion
      background-image: url("/images/burger.png") no-repeat center / contain;
```

---

Согласна,спасибо!
Только не "background-image", а просто "background".